packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -352,21 +429,26 @@ export default function makeKernelKeeper(kernelStorage, kernelSlog) {
 
   /**
    *
-   * @returns {number | 'never'}
+   * @returns { ReapDirtThreshold }
```

Comment:
I hate that we're inconsistent about presence vs. absence of whitespace inside JSDoc braces, but we currently are so no further comment here.

Reply:
which do you prefer?

Reply:
I don't have a strong opinion, but in terms of internal consistency it looks like "no spaces" wins by a lot so I guess that's the way to go:
```console
$ git grep '[*].*@[a-z].*[{].*[}]' packages/ \
  | grep -vE '@(import|see|link)' \
  | awk '{
    # Reduce $0 to `{…}`.
    gsub(/^[^*]*[*][^@]*@[a-z][^{]*|[^}]*$/, "");
    # Find the span [i, j) inside the wrapping braces and any adjacent spaces.
    i = match($0, /^[{] */) + RLENGTH;
    j = match($0, / *[}]$/);
    # 
    print substr($0, 1, i - 1) "..." substr($0, j, RLENGTH);
  }' \
  | sort \
  | uniq -c
    361 { ... }
     73 { ...}
     66 {... }
  10857 {...}
```